components/esp_websocket_client/esp_websocket_client.c

```diff
@@ -161,6 +161,21 @@ static esp_err_t esp_websocket_client_abort_connection(esp_websocket_client_hand
     return ESP_OK;
 }
 
+static esp_err_t esp_websocket_client_error_connection(esp_websocket_client_handle_t client)
+{
+    ESP_WS_CLIENT_STATE_CHECK(TAG, client, return ESP_FAIL);
+    esp_transport_close(client->transport);
+
+    if (client->config->auto_reconnect) {
+        client->wait_timeout_ms = WEBSOCKET_RECONNECT_TIMEOUT_MS;
```

Comment:
@0xFEEDC0DE64 Please remove this line, as `client->wait_timeout_ms` is already initialized either to default `WEBSOCKET_RECONNECT_TIMEOUT_MS` or to `config->reconnect_timeout_ms` during client initialization